src/dfx/src/actors/replica.rs

```diff
@@ -309,7 +309,7 @@ fn replica_start_thread(
             cmd.args(&["--http-port", &port.to_string()]);
         }
         if config.btc_adapter.enabled {
-            cmd.args(&["--subnet-features", "bitcoin_testnet"]);
```

Comment:
does bitcoin_regtest translates correctly into regtest string in the adapter config?

Reply:
This was already hard-coded here: https://github.com/dfinity/sdk/blob/9a377bc74e48db4063789108f47f33700155aa9d/src/dfx/src/lib/bitcoin/adapter/config.rs#L58